test/functional-api/integration/organisation/organisation.it-spec.ts

```diff
@@ -0,0 +1,143 @@
+import {
+  createOrganisationMutation,
+  deleteOrganisationMutation,
+  hostNameId,
+  organisationName,
+  updateOrganisationMutation,
+} from '@test/functional-api/integration/organisation/organisation.request.params';
+
+const legalEntityName = 'Legal alkemio';
+const domain = 'alkem.io';
+const website = 'alkem.io';
+const contactEmail = 'contact@alkem.io';
+
+describe('Organisation', () => {
+  let orgId = '';
+  beforeAll(async () => {
+    const res = await createOrganisationMutation(organisationName, hostNameId);
+    orgId = res.body.data.createOrganisation.id;
+  });
+  afterAll(async () => await deleteOrganisationMutation(orgId));
+
+  describe('create', () => {
+    test('should create', async () => {
+      const res = await createOrganisationMutation(
+        organisationName + '1',
+        hostNameId + '2',
+        legalEntityName,
+        domain,
+        website,
+        contactEmail
+      );
+
+      const data = res.body.data.createOrganisation;
+      const testOrgId = data.id;
+
+      expect(res.status).toBe(200);
+      expect(data).toMatchObject({
+        id: testOrgId,
+        displayName: organisationName + '1',
+        nameID: hostNameId + '2',
+        legalEntityName: legalEntityName,
+        domain: domain,
+        website: website,
+        contactEmail: contactEmail,
+      });
+
+      await deleteOrganisationMutation(testOrgId);
+    });
+
+    test('should FAIL on breaking unique nameID', async () => {
```

Comment:
The tests related ot unique nameID and displayDnameID, could be merged in a DDT.
[For reference](https://github.com/alkem-io/test-suites/blob/develop/test/functional-api/integration/opportunity/opportunity.it-spec.ts) test: `describe('DDT should not create opportunities with same nameID within the same challenge', () => {`

Reply:
I believe those should be in separate tests.
I agree that they cover the same type of functionality but for different fields and tests should be lean as possible